source/includes/entityref/_federaltaxform.md

```diff
@@ -0,0 +1,154 @@
+# FederalTaxForm
+
+This entity represents a tax form that is added to a candidate at the federal level.
```

Comment:
Can remove "This entity" and start with "Represents". I will be going thru and doing that generally at some point.